doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,374 @@
+.. _ug_nrf9160_gs:
+
+Getting started with nRF9160 DK
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section will get you started with your nRF9160 DK.
+You will update the firmware (both the application firmware and the modem firmware) and the nRF Cloud certificates of the DK, and conduct some initial tests.
+
+.. _nrf9160_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF9160 DK
+* nano-SIM card that supports :term:`LTE-M` or :term:`Narrowband Internet of Things (NB-IoT)` (the nRF9160 DK contains an iBasis SIM card)
+* micro-USB 2.0 cable
+
+Software
+========
+
+* Microsoft Windows 10
+* macOS X, latest version
+* Ubuntu Linux, latest Long Term Support (LTS) version
+
+.. _nrf9160_gs_installing_software:
+
+Installing the required software
+********************************
+
+To work with the nRF9160 DK firmware and certificates, install `nRF Connect for Desktop`_.
+
+After installing and starting the application, install the following apps:
+
+* Programmer
+* LTE Link Monitor
+
+.. _nrf9160_gs_updating_fw:
+
+Updating the DK firmware
+************************
+
+Download the latest application and modem firmware from the `nRF9160 DK Downloads <nRF9160 product website (compatible downloads)>`_ page and extract to a folder of your choice.
+
+The downloaded zip contains the following firmware:
+
+Application firmware
+  The :file:`img_app_bl` folder contains full firmware images for different applications.
+  When following this guide, use an image for the :ref:`asset_tracker_v2` application.
+  Asset Tracker v2 simulates sensor data and transmits it to Nordic Semiconductor's cloud solution, `nRF Cloud`_.
+
+  Depending on where you are located, you need the image with either LTE-M or NB-IoT support.
+  Check with your SIM card provider for the mode they support at your location.
+  For the iBasis SIM card provided with the nRF9160 DK, see `iBasis IoT network coverage`_.
+  If your location has support for both, you can choose either one.
+
+Application firmware for Device Firmware Update (DFU)
+  The images in the :file:`img_fota_dfu_bin` and :file:`img_fota_dfu_hex` folders contain firmware images for DFU.
+  When following this guide, you can ignore these images.
+
+Modem firmware
+  The modem firmware is in a zip archive instead of a folder.
+  The zip is named :file:`mfwnrf9160_` followed by the firmware version number.
+  Do not unzip this file.
+
+.. _nrf9160_gs_updating_fw_application:
+
+Updating the application firmware
+=================================
+
+To update the application firmware, complete the following steps.
+If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (Cmd+R on macOS), and try again.
```

Comment:
I think we should have same styling for Ctrl+R ..An earlier instance had ``Ctrl+ R``.

Reply:
Applied.